core/imt/inc/ROOT/TThreadExecutor.hxx

```diff
@@ -51,6 +51,8 @@ namespace ROOT {
       /// \endcond
       template<class F, class T>
       void Foreach(F func, std::vector<T> &args);
+      template<class F, class T>
```

Comment:
This commit is already in e538ae588

Reply:
yes, I just haven't rebased this branch on master yet :)